sysstat_report.py

```diff
@@ -260,7 +262,7 @@ def decompress(in_filepath: str, out_filepath: str) -> None:
             shutil.copyfileobj(in_file, out_file)
 
     @classmethod
-    def getSysstatDataFilepath(cls, date, filepath_formats: Sequence[str], temp_dir: str) -> None:
+    def getSysstatDataFilepath(cls, date, filepath_formats: Sequence[str], temp_dir: str) -> str or None:
```

Comment:
This return type annotation should be `Optional[None]` (although this was not introduced by your changes).

Reply:
Do you mean Optional[str]?

Reply:
Yes sorry, `Optional[str]`